crates/fluvio-smartengine/Cargo.toml

```diff
@@ -17,12 +17,12 @@ wasi = ["wasmtime-wasi", "engine"]
 transformation = ["serde_json", "serde_yaml"]
 default = ["engine"]
 
-
 [dependencies]
 tracing ={ workspace = true }
 thiserror = { workspace = true }
 anyhow = { workspace = true }
-serde = { workspace = true,  features = ['derive'] }
+# std is needed for ser/de AtomicU64
+serde = { workspace = true,  features = ['derive', 'std'] }
```

Comment:
previously `std` features is depended by `wasmtime`, I added it when developing WasmEdge